### DISCUSSION TITLE
Duplex SFP modules - how to connect cables to one port?

### DISCUSSION THREAD

**Discussion by mpwieczorek** [netbox-community/netbox#6220] [Q&A] [answered]
NetBox version - v2.11.0
Python version - 3.7

Hello, I have a configuration problem. How to connect 2 single mode optical cable to 1 duplex SFP module (switch interface)?

**Comment by candlerb** [comment, marked as answer]
https://groups.google.com/g/netbox-discuss/search?q=duplex

In short: you cannot connect two cables into one interface (and SFP modules are not modelled at all).

Option 1: treat each fibre pair as a single "cable".  This works fine if you are using duplex connections exclusively.
Option 2: connect just one leg to the ODF, and mark the partner port in the ODF as busy (connected) without actually connecting it.  This is a better option if you have mixed simplex and duplex fibres.  (The ability to do this was only just added in v2.11.0)
